src/pretix/presale/templates/pretixpresale/event/checkout_questions.html

```diff
@@ -105,9 +105,18 @@ <h4 class="panel-title">
                             {% for form in forms %}
                                 {% if form.pos.item != pos.item %}
                                     {# Add-Ons #}
-                                    <legend>+ {{ form.pos.item.name }}{% if form.pos.variation %} – {{ form.pos.variation.value }}{% endif %}</legend>
+                                    <legend> {# FIXME: I wanted to put it to the right side. For that legend should be d-flex and justify-content-between, but that's Bootstrap 4... #}
```

Comment:
I need help with the CSS here :sweat_smile:

Reply:
Putting the button before the actual text of the legend and replacing ``text-right flip`` with ``pull-right flip`` should be enough!